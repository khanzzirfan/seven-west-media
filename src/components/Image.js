import React from 'react';
import styled from 'styled-components';

const margin = 9;
const size = 38;

const Image = styled.img`
 margin-top:${margin}px;
 margin-left:${margin}px;
 width: ${size}px;
 height: ${size}px;
`;

export default Image;