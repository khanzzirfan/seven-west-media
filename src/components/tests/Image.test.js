import React from 'react';
import TestRenderer from 'react-test-renderer';
import 'jest-styled-components';

import Image from '../Image';

describe('components -> Image component', () => {
    it('should renders correctly', () => {
        const wrapper = TestRenderer.create(<Image src="media.png" alt="media image alt" />).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});