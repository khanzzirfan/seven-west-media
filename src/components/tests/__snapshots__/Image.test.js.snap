// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components -> Image component should renders correctly 1`] = `
.c0 {
  margin-top: 9px;
  margin-left: 9px;
  width: 38px;
  height: 38px;
}

<img
  alt="media image alt"
  className="c0"
  src="media.png"
/>
`;
